pkg/operator/sync.go

```diff
@@ -65,17 +65,14 @@ func (optr *Operator) syncClusterAPIController(config OperatorConfig) error {
 
 func (optr *Operator) waitForDeploymentRollout(resource *appsv1.Deployment) error {
 	return wait.Poll(deploymentRolloutPollInterval, deploymentRolloutTimeout, func() (bool, error) {
-		// TODO(vikas): When using deployLister, an issue is happening related to the apiVersion of cluster-api objects.
-		// This will be debugged later on to find out the root cause. For now, working aound is to use kubeClient.AppsV1
-		// d, err := optr.deployLister.Deployments(resource.Namespace).Get(resource.Name)
-		d, err := optr.kubeClient.AppsV1().Deployments(resource.Namespace).Get(resource.Name, metav1.GetOptions{})
+		d, err := optr.deployLister.Deployments(resource.Namespace).Get(resource.Name)
 		if apierrors.IsNotFound(err) {
 			return false, nil
 		}
 		if err != nil {
 			// Do not return error here, as we could be updating the API Server itself, in which case we
 			// want to continue waiting.
-			glog.Errorf("Error getting Deployment %q during rollout: %v", resource.Name, err)
+			glog.Errorf("Error vikas getting Deployment %q during rollout: %v", resource.Name, err)
```

Comment:
Can we don refer to Vikas specifically? :joy:

Reply:
Fame and fortune... :1st_place_medal: